src/AdminCommands.ts

```diff
@@ -0,0 +1,314 @@
+import { Logging } from "matrix-appservice-bridge";
+import * as yargs from "yargs";
+
+import { AdminCommand, ResponseCallback } from "./AdminCommand";
+import { Main } from "./Main";
+import { BridgedRoom } from "./BridgedRoom";
+
+const log = Logging.get("AdminCommands");
+
+const RoomIdDef = {
```

Comment:
This should be a type

Reply:
Huh, it's just a shorthand value for commands which take room ids as arguments. It has a value and can't be a type.

Reply:
it sure looks and sounds like a type... Can we call it something else that makes it sound like less of a type? Or refactor the code to make this not needed?

(I don't have a better name, but calling it a definition means it is screaming for an interface to implement).